hail/hail-ci-build.sh

```diff
@@ -44,29 +60,51 @@ trap on_exit EXIT
 # TERM is received ensures the EXIT handler is called.
 trap "exit 42" INT TERM
 
-GRADLE_OPTS=-Xmx2048m ./gradlew testAll makeDocs archiveZip --gradle-user-home /gradle-cache
+export GRADLE_OPTS="-Xmx2048m"
+export GRADLE_USER_HOME="/gradle-cache"
+
+echo "Compiling..."
+./gradlew shadowJar archiveZip > ${COMPILE_LOG}
+
+test_project() {
+    ./gradlew test > ${SCALA_TEST_LOG}
+    ./gradlew testPython > ${PYTHON_TEST_LOG}
+    ./gradlew doctest > ${DOCTEST_LOG}
```

Comment:
we should be able to run all of these in parallel too, right?

Reply:
how many system resources do these things have?

Reply:
and I don't think it will matter much -- this will still finish before VEP I think

Reply:
We're using n1-standard-8's and we reserve about half of the resources for each PR job. We could use bigger machines if it speeds up builds though.